sql/core/src/main/java/org/apache/spark/sql/execution/datasources/parquet/VectorizedColumnReader.java

```diff
@@ -93,13 +94,18 @@
   private final PageReader pageReader;
   private final ColumnDescriptor descriptor;
   private final OriginalType originalType;
+  // The timezone conversion to apply to int96 timestamps.  Null if no conversion.
+  private final TimeZone convertTz;
+  private final static TimeZone UTC = TimeZone.getTimeZone("UTC");
```

Comment:
I was avoiding doing that just because that makes the UTC vs GMT distinction more obvious :P.  I feel it may look odd having both defined as constants `DateTimeUtils`, without some comment explaining the difference, and if there is a preference.  Frankly I don't know the difference but I know impala uses UTC.